blog/2024-08-14-conda-ecosystem-explained.mdx

```diff
@@ -0,0 +1,63 @@
+
+## Conda Package Repository and Channels
+
+When it comes to managing packages with conda or conda-compatible tools, understanding where these packages come from is key. Anaconda.org serves as a central hub for conda packages, organized into various channels. While anyone can create their own channel on anaconda.org to host conda packages, we'll focus on the largest and most significant channels within the conda ecosystem.
+
+### `conda-forge`
+
+[conda-forge](https://conda-forge.org) is one of the largest and impactful channels in the conda ecosystem. Entirely community-driven, this channel hosts over 25,000 packages, making it a go-to resource for developers across the globe. Conda-forge is open-source, free to use, and maintained by a group of dedicated volunteers. The governance model ensures that the channel remains reliable and accessible to everyone.
+
+### Bioconda
+
+If your work involves computational biology, [Bioconda](https://bioconda.github.io) is the channel you’ll want to explore. Unlike conda-forge, which covers a wide variety of packages, Bioconda is focused on bioinformatics. It offers over 8,000 packages that cater to subdomains like genomics, proteomics, structural biology, and biomass analysis. Like conda-forge, bioconda is open-source and free to use, maintained by a community of volunteers.
+
+### `Defaults`
+
+The `defaults` channel is maintained by Anaconda, Inc. It's important to note that this channel comes with [Terms of Service](https://legal.anaconda.com/policies/en/). While largely free for individual users, students, and small companies, there are scenarios where a license might be required.
+
+## Getting Started with conda tools
+
+There are a few different ways to install and get started with conda:
+
+1. **Miniconda**: an installer developed by Anaconda, Inc. that installs conda and its dependencies. It is preconfigured to pull packages from the \`defaults\` channel, which has its own Terms of Service (as mentioned in conda channels section). You can always reconfigure conda after installation to use community channels like conda-forge or bioconda \[LINK to docs on how to do it\].
+2. **Miniforge**: developed by the conda-forge community, this installer comes with both conda and Mamba, and by default, it pulls packages from the conda-forge channel. It's a great option if you prefer a community-driven setup right from the start.
+3. **Micromamba:** developed by the Mamba developers at Quantstack, it’s a statically linked self-contained conda-like package and environment manager, distributed as a standalone executable without a preconfiguration of a default channel.
+4. **Pixi**: If you're looking for something a bit different, Pixi is a newer and opinionated conda-compatible package & environment manager written in Rust. Pixi is preconfigured to pull packages from the conda-forge channel.
```

Comment:
```
Pixi is preconfigured to pull packages from the conda-forge channel.
```

Is it? pixi has so many entrypoints that this can be confusing. Say, someone is migrating an environment.yml file with pixi but that file has defaults in its configuration, pixi won't remove it. There may be other corner cases but I'm not a heavy user of pixi to comment. I guess that someone with more experience should write on how to config channel in pixi and the possible pitfalls.